packages/42matters/api.js

```diff
@@ -15,10 +15,19 @@ class Api extends ApiKeyRequester {
             appleSearch: 'ios/apps/search.json',
             appleQuery: 'ios/apps/query.json',
             tencentLookup: 'tencent/android/apps/lookup.json',
-            amazonLookup: 'amazon/android/apps/lookup.json'
+            amazonLookup: 'amazon/android/apps/lookup.json',
+            sdks: 'sdks/search.json'
         }
         this.URLs = {}
         this.generateUrls();
+        // v1 schema json
+        this.v1BaseUrl = 'https://data.42matters.com/api/';
```

Comment:
I was thinking we could add all endpoints into the same object, and then have a root `v1` or `v2` to deliver the endpoints from. Like
```
this.endpoints = {
   v1: {
      baseUrl: '...',
      androidCountries: '...'
   },
   v2: {
      baseUrl: '...',
      accountStatus: '...'
   }
}
```

This could work better for organization and discovery, but we can iterate on it at a later time.

Reply:
I like that. I think that fits better than URLs, and doesn't have case confusion. Would be great to build that into the Requester type such that the developer did not need to worry about concatenating, nor about re-generating URLs if some base parameter changes.

Part of why I separated them is that the "v1" endpoints I don't think are documented and only get metadata. But, I think it would be fine to include them like above.